Lombiq.BaseTheme/Services/ResourceFilters.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Lombiq.BaseTheme.Constants;
+using Lombiq.HelpfulLibraries.OrchardCore.ResourceManagement;
+
+namespace Lombiq.DataTables.Navigation;
+
+[ResourceFilterThemeRequirement(FeatureIds.BaseTheme)]
+public class ResourceFilters : IResourceFilterProvider
+{
+    public void AddResourceFilter(ResourceFilterBuilder builder) =>
+        builder.Always().RegisterStylesheet(ResourceNames.Site);
+}
```

Comment:
How does using this filter stack up against the comment [here](https://github.com/Lombiq/Orchard-Base-Theme/pull/16/files#diff-137889361196787b5e675fd3837fb3a3d2786726567ddd5ee771b029aa1a51f2R14)?
> If you only have "Always" styles then it's easier to inject them via the Views/Widget-LayoutInjection.cshtml file and skip using a ResourceFilterMiddleware.

Reply:
I don't fully agree with that comment any more (past self is always a dumdum 😄) and I have reworded it:
> We always want to add the site style. If you only have "Always" styles and you are going to override Views/Widget-LayoutInjection.cshtml anyway, then it's easier to inject your style there and skip using a ResourceFilterMiddleware altogether.

Reply:
That makes sense. Sounds good like that.